### PR TITLE
Remove @ember/string (it's unused)

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
Supersedes: https://github.com/ember-cli/ember-cli-deprecation-workflow/pull/194

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -101,5 +100,6 @@
   "ember-addon": {
     "configPath": "tests/dummy/config",
     "after": "ember-cli-htmlbars"
-  }
+  },
+  "packageManager": "pnpm@9.6.0+sha512.38dc6fba8dba35b39340b9700112c2fe1e12f10b17134715a4aa98ccf7bb035e76fd981cf0bb384dfa98f8d6af5481c2bef2f4266a24bfa20c34eb7147ce0b5e"
```
Comment (NullVoxPopuli) [inline package.json:104]: this is added with newer versions of pnpm. 

As-is, there is nothing in the repo saying which pnpm version to use. 

CI says to use version 8 -- so I'll downgrade, set this, and set a volta config.

**Review by SergeAstapov — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -101,5 +100,10 @@
   "ember-addon": {
     "configPath": "tests/dummy/config",
     "after": "ember-cli-htmlbars"
+  },
+  "packageManager": "pnpm@8.15.9",
+  "volta": {
```
Comment (kategengler) [inline package.json:105]: We're adding volta?

Comment (NullVoxPopuli) [reply package.json:105]: it only matters for folks who have volta installed locally, and affects no one and nothing else

Comment (NullVoxPopuli) [reply package.json:105]: I set the tool versions to what CI is using

**Review by kategengler — APPROVED** [review]

**Pull request merged**